docs/hardware.rst

```diff
@@ -279,35 +279,70 @@ We currently recommend the Intel NUC for SecureDrop servers.
     could include: speakers, cameras, microphones, fingerprint readers,
     wireless, and Bluetooth cards.
 
-Intel NUC
-~~~~~~~~~
+.. _nuc7_recommendation:
+
+Intel 7th-gen NUC
+~~~~~~~~~~~~~~~~~
 
 The Intel NUC (Next Unit of Computing) is an inexpensive, quiet, low-power
 device that can be used for the SecureDrop servers. There are a
-`variety of models <https://www-ssl.intel.com/content/www/us/en/nuc/products-overview.html>`__
-to choose from. We recommend the
-`NUC5i5MYHE <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc5i5myhe.html>`__
-because it has a mid-range CPU (the 5th generation Intel i5), a Mini
-DisplayPort port for a monitor, and two USB 3.0 ports for faster OS
-installation and data transfer.
+`variety of models <https://www.intel.com/content/www/us/en/products/boards-kits/nuc.html>`__
+to choose from. 
+
+The NUCs typically come as kits, and some assembly is required. You will need to
+purchase the RAM and hard drive separately for each NUC and insert both into the
+NUC before it can be used. We recommend:
+
+-  2x 240GB HDDs (2.5")
+-  1x memory kit of compatible 2x4GB sticks
+   -  You can put one 4GB memory stick in each of the servers.
+
+.. note:: We recommend the use of HDDs over SSDs, as it's more difficult to 
+          securely delete data from SSDs.
+
+We have tested and can recommend the `NUC7i5BNH <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc7i5bnh.html>`__ - these tend to be readily available in 
+retail stores.
+
+The NUC7i5BNH has soldered-on wireless components, which cannot easily be
+removed. For security reasons, we recommend that you take the following steps
+to disable wireless functionality:
+
+- before installation of the RAM and storage, disconnect the wireless antennae 
+  leads.
+
+|NUC7 leads|
+
+- before the initial OS installation, boot into the BIOS by pressing **F2** at
+  startup, navigating to **Advanced > Devices > Onboard Devices**, and disabling
+  unwanted hardware - everything except **LAN**.
+
+|Visual Bios|
+  
+
+.. |NUC7 leads| image:: images/hardware/nuc7-leads.jpg
+.. |Visual BIOS| image:: images/hardware/visualbios.png
+
+Other 7th-generation NUCs have also been reported to work, although we have not 
+tested them. For example, the `NUC7i5DNHE <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc7i5dnhe.html>`__ uses the same Ethernet chipset as the NUC7i5BNH,
+and also has a removable wireless card, simplifying the server setup process. 
+However, it may be harder to find a retail source for this model.
+
+Previous Server Recommendations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Intel 5th-gen NUC
+~~~~~~~~~~~~~~~~~
+
+We previously recommended the
+`NUC5i5MYHE <https://www.intel.com/content/www/us/en/products/boards-kits/nuc/kits/nuc5i5myhe.html>`__, however, it is now EOL, and difficult to find. We will continue to support and
```

Comment:
Nit: is now EOL -> has reached end-of-life (just for clarity for folks less familiar with terms like "EOL")